internal/configs/mock_provider.go

```diff
@@ -393,12 +450,20 @@ func decodeOverrideDataBlock(block *hcl.Block, source OverrideSource) (*Override
 	return override, diags
 }
 
+var (
+	// When this attribute is set to true, the values specified in the override
+	// block will be used for computed attributes even when planning. Otherwise,
+	// the computed values will be set to unknown, just like in a real plan.
+	overrideComputed = "override_computed"
```

Comment:
We have the `command` attribute available in the run blocks: https://developer.hashicorp.com/terraform/language/tests#run-blocks

I wonder if like `compute_command` that maps to a keyword of either `plan` or `apply` might be more clear about what is happening here? We can make it default to `apply`. 

I think that makes it a bit more clear about what the attribute is actually doing. What do you think?

Reply:
Thanks for the suggestion. I contemplated quite a bit on the right name to communicate what's going on here, and you are right that `override_computed` seems a bit unclear. However, I have similar thoughts about `compute_command`, especially the `compute` part. That may be confusing. What about `override_command/override_target = plan/apply`? From a user POV, they have defined the override, and want it to be applied. That it is a computed attribute is more subtle.

Reply:
I like `override_command`, I think that could work. Another idea I had was `override_during` - as an attribute it doesn't make much sense on it's own, but when coupled with `apply` or `plan` then it brings a lot of context eg. `override_during = plan`.

But, `override_command` is also good I think 👍